docs/book/src/clusterctl/overview.md

```diff
@@ -167,8 +167,8 @@ The `clusterctl init` command installs the Cluster API components and transforms
 into a management cluster.
 
 The command accepts as input a list of providers to install; when executed for the first time, `clusterctl init` 
-automatically adds to the list the Cluster API core provider, and if a bootstrap provider is not specified, it adds
-also the kubeadm bootstrap provider.
+automatically adds to the list the `cluster-api` core provider, and if not specified, it adds
+also the `kubeadm-bootstrap` provider and the `kubeadm-controlplane` provider.
```

Comment:
Potential readability improvement if you drop `adds` from the previous line, WDYT?
```suggestion
also adds the `kubeadm-bootstrap` and `kubeadm-controlplane` providers.
```